packages/cli/src/models/local/ContractManager.ts

```diff
@@ -51,11 +53,13 @@ export default class ContractManager {
   }
 
   private isLocalContract(
-    buildDir: string,
-    contract: { [key: string]: any },
+    contractsDir: string,
+    contract: { sourcePath: string },
+    root: string,
   ): boolean {
-    const projectDir = buildDir.replace('build/contracts', '');
-    return contract.sourcePath.indexOf(projectDir) === 0;
+    const cwd = root || process.cwd();
```

Comment:
nit: you can define `process.cwd()` as the default value in the `isLocalContract` function definition